### ISSUE TITLE
During `vagrant provision`, Ansible fails when username & site domain are too long

### ISSUE THREAD

**Issue by jdsteinbach** [evolution/wordpress#138]
## Problem:
When the OS X username & site domain are too long, they cause an ansible error during `vagrant provision`:

```sh
fatal: [local.innovativemedianetwork.com] => SSH Error: unix_listener: "/Users/jamessteinbach/.ansible/cp/ansible-ssh-local.innovativemedianetwork.com-22-vagrant.tRI3QuUSacSXgiA6" too long for Unix domain socket
    while connecting to 192.168.161.53:22
It is sometimes useful to re-run the command using -vvvv, which prints SSH debug output to help diagnose the issue.
```

## Solution:
Add the following to `ansible.cfg`:

```py
[ssh_connection]
control_path = %(directory)s/%%h-%%p-%%r
```
Ansible issue: https://github.com/ansible/ansible/issues/11536

**Comment by jdsteinbach** [comment]
Also, if this file is written automatically by `yo evolve wordpress` then the Yeoman generator will need the same update.

**Comment by EvanK** [comment]
We already have a FAQ entry for this (see 6ef7b92). I suppose we could use the longer `%h-%p-%r` pattern that's widely supported across ssh versions, and then update the faq to suggest `%C` when the issue persists (and the user's ssh version supports it)

**Comment by EvanK** [comment]
This is fixed, as of 1.4.1
